torchtune/training/checkpointing/_checkpointer.py

```diff
@@ -399,6 +400,10 @@ class FullModelHFCheckpointer(_CheckpointerInterface):
             Default is True.
         should_load_recipe_state (bool): If True, the checkpointer will load the additional checkpoint files corresponding to
             the receipe state from a previous run. Default is False
+        enable_dcp (bool): If True, the checkpointer will load the checkpoint file using dcp checkpointing apis.
```

Comment:
Can you add a comment just saying this is experimental for now? We may change up the param while keeping the same core code.